grpc/alteration_percent_test.go

```diff
@@ -0,0 +1,582 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package grpc_test
+
+import (
+	"fmt"
+
+	. "github.com/DataDog/chaos-controller/grpc"
+	pb "github.com/DataDog/chaos-controller/grpc/disruption_listener"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("get mapping from randomly generated Percent to Alteration based on AlterationToPercentAffected", func() {
```

Comment:
If they have the same `Describe`, how are `alteration_percent_test.go` and `percent_to_alteration_test.go` different?

Reply:
okay I now see how they're different, but it does seem like at least one of them needs a different `Describe` text